src/app/containers/FrontPageSection/index.jsx

```diff
@@ -22,6 +23,10 @@ import idSanitiser from '#lib/utilities/idSanitiser';
 
 // Apply the right margin-top to the first section of the page when there is one or multiple items.
 const FirstSectionTopMargin = styled.div`
+  margin-top: ${GEL_SPACING};
```

Comment:
Currently on mobile, the margin at the top of the page is on the grid, but on desktop the margin on the grid is 0, and is added instead to the top story. This made positioning the title across breakpoints tricky. For now I have removed margin from top of the grid and added it to the top story section on mobile as well, but might be nicer to do the other way round (which requires more refactoring)?

Reply:
When you say `For now I have removed margin from top of the grid` where exactly is that?

Reply:
There is a `GridItemConstrainedLargeWithTopMargin` component as well as a `GridItemConstrainedLarge` component - I used `GridItemConstrainedLarge` instead of `GridItemConstrainedLargeWithTopMargin`

Reply:
Ok, you have used another Grid component. I initially thought you removed the margin-top from the previous one 👍